.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:lts
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            # Find a cache corresponding to this specific package-lock.json checksum
+            # when this file is changed, this key will fail
+            - v1-npm-deps-{{ checksum "package-lock.json" }}
+            # Find the most recently generated cache used from any branch
+            - v1-npm-deps-
+      - run:
+          name: Install Dependencies
+          command: npm ci
+
+      - run:
+          name: Install JUnit coverage reporter
+          command: npm install --save-dev jest-junit
+
+      - save_cache:
+          key: v1-npm-deps-{{ checksum "package-lock.json" }}
+          paths:
+            - ~/.npm
+
+      - run:
+          name: Run test
+          command: npm run test
+
+      - run:
+          name: Save test report
+          command: |
+            mkdir -p ./reports/junit
+            find . -type f -regex ".*/junit.xml" -exec cp {} ./reports/junit/ \;
+          when: always
+
+      - store_test_results:
+          path: ./reports/junit/
+
+      - store_artifacts:
+          path: ./reports/junit
+
+      - store_artifacts:
+          path: dist/coverage
```

Comment:
let's have a single dist output folder for everything! (i.e move reports/ to dist/ ?)